src/services/tasks.service.ts

```diff
@@ -8,4 +9,70 @@ export const getTaskByStatus = async (status: string): Promise<Task[]> => {
     return await Task.findAll({
         where: { status }
     });
+}
+
+export const getTaskByID = async (id: string): Promise<Task> => {
+    try{
+        const task = await Task.findOne({
+            where: { id }
+        });
+
+        if(task !== null){
+            return task;
+        }else{
+            throw('Could not find task ID in database');
+        }   
+    }catch(err){
+        return(err);
+    }
+}
+
+export const addTask = async (task: string): Promise<string> => {
+    const newTask = await Task.create({task: task, status: ETaskStatus.Incomplete});
+    return(newTask.id);
```

Comment:
What will happen if we want to retrieve of do something else with the new created task? We need to fetch the task by id again in the upper-level function right? 

The function here is correct but the logic seems can be updated. I think we can return new created Task, then the upper-level function will decide what to return out.